sympy/parsing/latex/lark/latex.lark

```diff
@@ -243,6 +247,18 @@ list_of_expressions: _expression ("," _expression)*
 
 function_applied: _oneletter_symbol L_PAREN list_of_expressions R_PAREN
 
+min: FUNC_MIN L_PAREN list_of_expressions R_PAREN
+
+max: FUNC_MAX L_PAREN list_of_expressions R_PAREN
+
+bra: CMD_LANGLE _atomic_expr BAR
+
+ket: BAR _atomic_expr CMD_RANGLE
+
+inner_product: CMD_LANGLE _atomic_expr BAR _atomic_expr CMD_RANGLE
```

Comment:
This makes sense.

Reply:
Good point, I didn't think of the case you mentioned. The main thing I wanted to prevent was something like $| \cos x \rangle$ which is meaningless. `_expression` allows for strings like that, but I guess that's something we'll have to live with.

Reply:
SymPy can sort out the things that doesn't make sense. If the SymPy objects don't validate the input type, it's usually their own issue.